File: Matlab/int-ai.m
% Define the function
sin_function = @(x) sin(x);

% Define the integration limits
lower_limit = -pi;
upper_limit = 2/3 * pi;

% Compute the definite integral using quad function
area = integral(sin_function, lower_limit, upper_limit);

fprintf('Area between -π and 2/3π for sin(x) = %.6f\n', area);

File: python/int-ai.py
import numpy as np
from scipy.integrate import quad

# Define the function
def sin_function(x):
    return np.sin(x)

# Define the integration limits
lower_limit = -np.pi
upper_limit = 2/3 * np.pi

# Compute the definite integral
area, _ = quad(sin_function, lower_limit, upper_limit)

print("Area between -π and 2/3π for sin(x) =", area)
